Analysis/ScanLibraries/source/XiaListModeDataMask.cpp

```diff
@@ -37,14 +37,10 @@ FIRMWARE XiaListModeDataMask::ConvertStringToFirmware(const std::string &type) {
         firmware = R30980;
     else if (firmwareNumber >= 30981 && firmwareNumber < 34688)
         firmware = R30981;
-    else if (firmwareNumber ==
-             34688) //compare exactly here since nothing higher
+    else if (firmwareNumber >= 34688)
```

Comment:
Wouldn't it be nice to have a warning if a firmware was specified that may be beyond the last known value?

Reply:
Opened issue #41.

Reply:
Would it be nice? I guess so since I see the slack notification for the ticket. I think that it's a good call out.